nilearn/_utils/ndimage.py

```diff
@@ -25,6 +25,11 @@ def largest_connected_component(volume):
     volume: numpy.array
         3D boolean array with only one connected component.
     """
+    if hasattr(volume, "get_data") \
+       or isinstance(volume, _basestring):
```

Comment:
Style: Python developers do not like continuation lines. We prefer to use parenthesis. We will merge this, to avoid more waiting time, but change it after merge.